### ISSUE TITLE
Showing the downloader under playing 

### ISSUE THREAD

**Issue by tanja84dk** [Seklfreak/discord-image-downloader-go#45]
I have a small issue and that that the account is showing data from the downloader as if its playing that game like at this moment its saying "Playing 7 pictures downloaded".

Is it posible to get that removed in some way because I'm thinking som admins would maybe kick and ban people for that

**Comment by Kakkela** [comment]
Personally I think it's common courtesy to show that you might be downloading images from the server. If Admins don't like it you shouldnt be doing it under discuise either.

And if you are using it on main account and think someone wont like it on server you arent really downloading in, I would suggest moving the downloader to second account and only join on servers you download from.

It's fairly simple to get out of the code and build the bot yourself (or try to submit pull request), but I doubt @Seklfreak is going to change the behaviour.

**Comment by tanja84dk** [comment]
I'm not using my main account but using my uploader account there is verified some places to upload with because If I dont want the upload to be linked with me even that all I upload is legal

**Comment by Kakkela** [comment]
Could you rephrase that, I don't quite understand what you are trying to say.

**Comment by sxnfxsed** [comment]
The code that shows that you're downloading pictures is [here](https://github.com/Seklfreak/discord-image-downloader-go/blob/master/main.go#L1595). You can change it or remove it entirely to stop the program from showing the status. From there you can compile it or just run it from the source in console. It's not too difficult, I had no go experience prior to downloading this.

**Comment by DrakeTDL** [comment]
you may get an error while build so be weary

**Comment by EvilWalks** [comment]
@sxnfxsed if you see this, mind explaining real quick how you'd go about running it from source in the console? I'd prefer to have it not show download count, as it keeps over writing whatever game I'm playing, or music I'm listening to, when I'd prefer to be sharing those activities instead.

Thanks.

**Comment by DrakeTDL** [comment]
@EvilWalks do go run to run it from console.

**Comment by DreamPhreak** [comment]
Personally what I did was export (or is the correct term "compile"?) into a .exe file for portability to run it from another computer without having to install Go lang onto that computer.

DrakeTDL's is correct too.

**Comment by EvilWalks** [comment]
I just get errors when trying to run or build it. So I might have to come back to this another day, when I've managed to work out what I'm doing.

**Comment by DrakeTDL** [comment]
replace "github.com/mvdan/xurls" on line 27 with "mvdan.cc/xurls"

**Comment by tanja84dk** [comment]
I guess its imposible to build/compile the source anymore when we dont have the versions there is used at start.

Because when I now tried to compile it after I had changed the xurls line in import, then I get this error after

# command-line-arguments
.\main.go:482:28: xurls.Strict.FindAllString undefined (type func() *regexp.Regexp has no field or method FindAllString)
.\main.go:657:35: xurls.Strict.FindAllString undefined (type func() *regexp.Regexp has no field or method FindAllString)
.\main.go:724:31: xurls.Strict.FindAllString undefined (type func() *regexp.Regexp has no field or method FindAllString)

**Comment by tanja84dk** [comment]
and I have tried out from the source files for the newest release

**Comment by DreamPhreak** [comment]
Found a fix for this which I also added to my fork.

Where it says xurls.Strict.FindAllString, add () after Strict so you'll end up with xurls.Strict().FindAllString

Found this when going through xurls documentation where they explain Strict(), Relaxed(), and something, noticed it was missing, added it, builds without any problems.

Don't have to take out huge chunks of code anymore.
